samples/CallWithChat/public/index.html

```diff
@@ -22,8 +22,14 @@
       document.head.appendChild(script);
     </script>
     <style>
-      body { display: flex; flex-direction: column; height: auto; }
-      #Root { flex: 1; }
+      body {
```

Comment:
Are these changes needed?

Reply:
I did not make this change. I believe this happened because rush prettier runs every time I commit